Gemfile

```diff
@@ -85,7 +85,7 @@ group :test, :development do
   # Pretty printed test output
   gem 'turn', '~> 0.8.3', :require => false
   gem 'fuubar'
-  gem 'rspec-rails'
+  gem 'rspec-rails', '~> 2.99.0'
```

Comment:
I had to revert back to the non-upgraded version of the gems to get things to run on my system.... But I know Steve mentioned having troubles getting running on his system, hence the upgrades.

This is something that might draw up some issues going forward. So yeah, what Rob said... what was the issue that prompted the upgrade?